gpii/node_modules/dbOperation/index.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+var fluid = require("infusion");
+
+fluid.module.register("gpii-db-operation", __dirname, require);
```

Comment:
I meant by the rename for the entire module to be renamed - the main purpose of the rename is for its name in package.json to be a valid npm module name. This directory, and the name in package.json should be renamed as well. We can retain the camel-casing in the individual file names.